<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileBox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/images/images.qrc">
    <normaloff>:/images/FileBox_1.png</normaloff>:/images/FileBox_1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-size: 18px;
	font-family: Trajan;
	color: black;
}

#pbRegister{
	border-radius: 15px;
	background: red;
	color: white;
}

QPushButton:hover{
	border-radius: 5px;
	background: #800000;
	color: #333;
}

#pbRegister:hover{
	border-radius: 15px;
	background: #800000;
	color: #333;
}

#frame{
	background: #333;
	border-radius: 20px;
}

QPushButton{
	border-radius: 5px;
	background: red;
	color: white;
}

QLineEdit{
	background: transparent;
	border: none;
	color: #717072; 
	border-bottom: 1px solid #717072;
}

#lblRegister{
	color: white;
	font-size: 26px;
}

#Register{
	background-image: url(:/images/bg.png);
}

#lblWarning{
	color: red;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>480</width>
     <height>520</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>520</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>520</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pbRegister">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>REGISTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUsername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leConfirmPassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRegister">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWarning">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leIP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>IP Address</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
